sdk/identity/identity/test/public/node/clientCertificateCredential.spec.ts

```diff
@@ -30,19 +30,14 @@ describe("ClientCertificateCredential", function () {
     await cleanup();
   });
 
-  const certificatePath = path.join(ASSET_PATH, "fake-cert.pem");
+  const certificatePath = env.IDENTITY_SP_CERT_PEM || path.join(ASSET_PATH, "fake-cert.pem");
   const scope = "https://vault.azure.net/.default";
 
   it("authenticates", async function (this: Context) {
-    if (isLiveMode()) {
-      // Live test run not supported on CI at the moment. Locally should work though.
-      this.skip();
-    }
-
     const credential = new ClientCertificateCredential(
-      env.AZURE_TENANT_ID!,
-      env.AZURE_CLIENT_ID!,
-      certificatePath!,
+      env.IDENTITY_SP_TENANT_ID || env.AZURE_TENANT_ID!,
+      env.IDENTITY_SP_CLIENT_ID || env.AZURE_CLIENT_ID!,
+      env.IDENTITY_SP_CERT_PEM || certificatePath!,
```

Comment:
We really should work on reducing the overlap of environment variables: https://github.com/Azure/azure-sdk-for-js/issues/21056